src/order/bounded.lean

```diff
@@ -0,0 +1,428 @@
+/-
+Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import .min_max
+import .rel_classes
+import data.set.intervals.basic
+
+/-!
+# Bounded and unbounded sets
+
+We prove miscellaneous lemmas about bounded and unbounded sets. Many of these are just variations on
+the same ideas, or similar results with a few minor differences. The file is divided into these
+different general ideas.
+-/
+
+variables {α : Type*} {r : α → α → Prop} {s t : set α}
+
+/-! ### Subsets of bounded and unbounded sets -/
+
+theorem bounded_of_subset_bounded (hst : s ⊆ t) (hs : bounded r t) : bounded r s :=
+begin
```

Comment:
I think you can golf this with `hs.imp $ λ ha b hb, ha b (hst hb)` or something similar

Reply:
Just golfed a large section of the file.